src/main/webapp/script.js

```diff
@@ -58,4 +59,32 @@ async function displayDailyDeed() {
 
     deedTitle.innerText = daily_deed.title;
     deedDescription.innerText = daily_deed.description;
+}
+
+/**
+ *displays the current date using the built in Date() class
+ */
+function displayDate() {
+    const current_date = document.getElementById("curDate");
+
+    var months = new Object();
+    months[1] = "January";
+    months[2] = "February";
+    months[3] = "March";
+    months[4] = "April";
+    months[5] = "May";
+    months[6] = "June";
+    months[7] = "July";
+    months[8] = "August";
+    months[9] = "September";
+    months[10] = "October";
+    months[11] = "November";
+    months[12] = "December"; 
```

Comment:
Can these Strings be turned in class constants?

Reply:
Yes that's a good idea!